src/main/java/com/googleinterns/zoomtube/data/TranscriptLine.java

```diff
@@ -26,28 +27,31 @@
   public static final String PROP_START = "start";
   public static final String PROP_DURATION = "duration";
   public static final String PROP_CONTENT = "content";
+  public static final String PROP_END = "end";
 
   /**
    * Creates a {@code TranscriptLine} object.
    * @param key The key for the transcript.
    * @param lecture The key for the lecture.
-   * @param start The starting timestamp for the lecture line in seconds.
+   * @param start The starting time for the lecture line in seconds.
    * @param duration The number of seconds that the timestamp lasts for.
    * @param content The text content of the transcript line.
    */
   // TODO: Update start to be a Date object and duration to be a long.
   public static TranscriptLine create(
-      Key key, Key lecture, String start, String duration, String content) {
-    return new AutoValue_TranscriptLine(key, lecture, start, duration, content);
+      Key key, Key lecture, Date start, double duration, Date end, String content) {
+    return new AutoValue_TranscriptLine(key, lecture, start, duration, end, content);
   }
 
   public abstract Key key();
 
   public abstract Key lecture();
 
-  public abstract String start();
+  public abstract Date start();
 
-  public abstract String duration();
+  public abstract double duration();
```

Comment:
Can duration be `long` milliseconds, or a `Date`?

Reply:
I think this is preferred to be Date

Reply:
Sounds good